app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#6200EE</color>
-    <color name="colorPrimaryDark">#3700B3</color>
-    <color name="colorAccent">#03DAC5</color>
+    <color name="colorPrimary">#00B0FF</color>
+    <color name="colorPrimaryLight">#69E2FF</color>
+    <color name="colorPrimaryDark">#0081CB</color>
+    <color name="colorAccent">#0080FF</color>
+
+    <color name="colorSecondary">#FFEE58</color>
+    <color name="colorSecondaryLight">#FFFF8B</color>
+    <color name="colorSecondaryDark">#EDCB43</color>
+
+    <color name="colorTextLight">#FFFFFF</color>
+    <color name="colorTextDark">#000000</color>
+
+    <color name="colorIconLight">#FFFFFF</color>
+    <color name="colorIconDark">#666666</color>
+
+    <color name="colorSuccess">#00B85C</color>
+    <color name="colorError">#FF8282</color>
 </resources>
```

Comment:
new line

Reply:
Idk about this tbh (Having new line for every file, is that common for you guys)
Other than that it LGTM

Reply:
I wasn't whether to add it or not since the file was initialized without one

Reply:
I see! It's funny how the workflow is not checking for new line as well. Might need to edit the settings

Reply:
I'll add a new line and we'll use that as the convention

Reply:
changed noticed! Resolved.